frame/support/src/migrations.rs

```diff
@@ -209,3 +211,69 @@ impl<P: Get<&'static str>, DbWeight: Get<RuntimeDbWeight>> frame_support::traits
 		Ok(())
 	}
 }
+
+/// A migration that can proceed in multiple steps.
+pub trait SteppedMigration {
+	fn id(&self) -> BoundedVec<u8, ConstU32<32>>;
```

Comment:
Vec is also fine here, but I have usually seen types like `[8u; 32]` being used for such cases.

Reply:
I also made it generic. But with the new default, we could add some sane default like this :thinking: